src/tests/miscellaneous/mock-adb/app/bin.js

```diff
@@ -92,6 +96,10 @@ async function main() {
 
     result.input = process.argv;
     result.inputCommand = inputCommand;
+
+    fs.writeFileSync(path.join(testLogsDir, 'adb.log'), `ADB ${inputCommand}\n`, {
```

Comment:
Might consider EOL instead of '\n'
```suggestion
    fs.writeFileSync(path.join(testLogsDir, 'adb.log'), `ADB ${inputCommand}` + EOL, {
```

Reply:
Since we expect this file to be compared to a snapshot in our E2E tests, we don't want different EOL characters on different OSs.